tests/conftest.py

```diff
@@ -6,7 +6,7 @@
 from qibolab import PLATFORMS, create_platform
 
 ORIGINAL_PLATFORMS = os.environ.get(PLATFORMS, "")
-DUMMY_PLATFORM_NAMES = ["qm", "qblox", "rfsoc", "zurich"]
+DUMMY_PLATFORM_NAMES = ["dummy_couplers", "qm", "qblox", "rfsoc", "zurich"]
```

Comment:
There are possibly too many dummies here (lhs & rhs).

However, dummy on the rhs is fine, the left one is dummy in a different sense, so it has been badly chosen (most likely by me).

But I'd try to avoid the `_couplers`: do we really need to maintain a dummy w/ and w/o couplers?

(This comment is quite unrelated to this PR)

Reply:
The original idea was that having a dummy with couplers should be sufficient for testing both so initially we introduced it on top of the existing dummy. However, then we were testing (mainly qibocal) using only the dummy with couplers and we ended up breaking multiple routines for platforms that did not have couplers without noticing. So we decided to create a dummy for both cases to be safe.

Most likely with a better design of `Qubit`/`Coupler` objects we will be able to get away with one dummy, but until we reach that stage, I'd prefer to keep both.

Regarding lhs, I don't see any issue with the name. Maybe we could just document that these are the platforms used in the CI tests that have no access to instruments and they represent artificial configurations that could exist in a lab (but do not exist in the real lab).

Reply:
> Regarding LHS, I don't see any issue with the name. Maybe we could just document that these are the platforms used in the CI tests that have no access to instruments and they represent artificial configurations that do not exist in the real lab (although in principle they could).

Yeah, but we could call them `TESTING_PLATFORM_NAMES` or `SANDBOX_PLATFORM_NAMES` or whatever else.
My idea here is that is confusing talking about a `dummy-qm` and a `dummy-dummy`, because the word dummy is used both to signal that we're using them for CI tests, but also for the specific dummy platform.

Reply:
Concerning everything else: thanks for the explanation, I missed part of the story. Then I agree with you, let's postpone the possible unification to after the couplers reshuffling.